src/e2e/java/teammates/e2e/cases/lnp/FeedbackSessionSubmitLNPTest.java

```diff
@@ -0,0 +1,273 @@
+package teammates.e2e.cases.lnp;
+
+import java.io.IOException;
+import java.time.Instant;
+import java.time.ZoneId;
+import java.util.ArrayList;
+import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.Map;
+
+import org.apache.jmeter.protocol.http.control.HeaderManager;
+import org.apache.jorphan.collections.HashTree;
+import org.apache.jorphan.collections.ListedHashTree;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+
+import teammates.common.datatransfer.DataBundle;
+import teammates.common.datatransfer.FeedbackParticipantType;
+import teammates.common.datatransfer.InstructorPrivileges;
+import teammates.common.datatransfer.attributes.AccountAttributes;
+import teammates.common.datatransfer.attributes.CourseAttributes;
+import teammates.common.datatransfer.attributes.FeedbackQuestionAttributes;
+import teammates.common.datatransfer.attributes.FeedbackSessionAttributes;
+import teammates.common.datatransfer.attributes.InstructorAttributes;
+import teammates.common.datatransfer.attributes.StudentAttributes;
+import teammates.common.datatransfer.questions.FeedbackQuestionDetails;
+import teammates.common.datatransfer.questions.FeedbackTextQuestionDetails;
+import teammates.common.util.Const;
+import teammates.e2e.util.BackDoor;
+import teammates.e2e.util.JMeterElements;
+import teammates.e2e.util.LNPTestData;
+
+/**
+ * L&P Test Case for instructor's student enrollment API endpoint.
```

Comment:
Forget to change?

Reply:
Updated javadoc to reflect on the actual test.